www-apps/hugo/hugo-0.92.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module bash-completion-r1
+
+DESCRIPTION="The world's fastest framework for building websites"
+HOMEPAGE="https://gohugo.io https://github.com/gohugoio/hugo"
+
+SRC_URI="
+	https://github.com/gohugoio/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://tastytea.de/files/hugo-0.92.2_deps.tar.xz
+"
+
+# NOTE: To create dependency tarball:
+#       Run `go mod vendor && tar -cJf ${P}_deps.tar.xz vendor`
```

Comment:
`cd .. && tar -cJf ${P}-vendor.tar.xz ${P}/vendor` and then you won't need to override `src_unpack`.